magenta/interfaces/midi/midi_hub.py

```diff
@@ -166,8 +168,21 @@ def __init__(self,
     self._duration = duration
     # A signal for when to stop the metronome.
     self._stop_time = stop_time
+    self._qpb = qpb
+    self._count = 0
+    # The amount to add to metronome velocity on downbeats only.
+    self._accent_velocity_delta = 10
+    # The pitch interval in semitones to raise metronome downbeats only.
+    self._accent_pitch_interval = 7
     super(Metronome, self).__init__()
 
+    # limit metronome accent velocity to 127 max value.
+    self._accent_velocity = min(velocity + self._accent_velocity_delta, 127)
+    # only raise metronome accent pitch if there is headroom.
```

Comment:
the default is set to a 5th which is pretty neutral. If for some reason somebody wanted the pitch to be at the top of the keyboard, this at least prevents it from being forced to something that might be more harmonically colored like a major/minor third or tritone.